src/main/java/org/broadinstitute/consent/http/configurations/MailConfiguration.java

```diff
@@ -8,13 +8,15 @@
 public class MailConfiguration {
 
     @NotNull
-    boolean activateEmailNotifications;
+    private boolean activateEmailNotifications;
 
     @NotNull
-    String googleAccount;
+    private String googleAccount;
 
     @NotNull
-    String sendGridApiKey;
+    private String sendGridApiKey;
+
+    private String sendGridStatusUrl;
```

Comment:
For consistency, we should make this not null as well
```suggestion
    @NotNull
    private String sendGridStatusUrl;
```